src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -33,8 +35,18 @@ private static void setupPresetTests(Application application) {
 		SelectTestFigure2OptionListener selectTestFigure2OptionListener = new SelectTestFigure2OptionListener(
 				DriverFeature.getDriverManager());
 
+
+		ActionListener rectangle = e -> ComplexCommandUsage.drawRectangle(0,0, 50, 25).execute(DriverFeature.getDriverManager().getCurrentDriver());
+
+		ActionListener triangle = e -> ComplexCommandUsage.drawTriangle(-50,-50, 50, 50).execute(DriverFeature.getDriverManager().getCurrentDriver());
```

Comment:
It is good to keep a convention used in the original code (class instance instead of lambda). 
It is possible to rewrite all tests to the same convention, but I would prefer avoiding such long one-liners (https://pl.wikipedia.org/wiki/Prawo_Demeter).